vizro-core/src/vizro/models/_navigation/nav_bar.py

```diff
@@ -48,9 +50,10 @@ def pre_build(self):
         ]
 
         for position, item in enumerate(self.items, 1):
-            # The filter icons are named filter_1, filter_2, etc. up to filter_9. If there are more than 9 items, then
-            # the 10th and all subsequent items are named filter_9+.
-            item.icon = item.icon or f"filter_{position}" if position <= 9 else "filter_9+"  # noqa: PLR2004
+            # The default icons  (if none are specified) are named filter_1, filter_2, etc. up to filter_9.
+            # If there are more than 9 items, then the 10th and all subsequent items are named filter_9+.
+            icon_default = "filter_9+" if position > MAX_NUMBER_DEFAULT_ICONS else f"filter_{position}"
```

Comment:
```suggestion
            icon_default = "filter_9+" if position > 9 else f"filter_{position}"
```

I don't think there's any point extracting this into a constant because the `9+` is hardcoded into the same line, so introducing `MAX_NUMBER_DEFAULT_ICONS§ really just increases indirection without making anything more general.

Reply:
I was trying to get rid of a linting error that's otherwise raised by ruff 😄 You would prefer to have the ignore instead then?

Reply:
Oh I see. Yes, we're better than ruff 😀 So ignore it.